libres/lib/analysis/enkf_linalg.cpp

```diff
@@ -35,10 +35,26 @@ void enkf_linalg_genX3(matrix_type *X3, const matrix_type *W,
     matrix_free(X2);
 }
 
+/**
+ * Implements a version of Eq. 14.34 from the book Data Assimilation,
+ * The Ensemble Kalman Filter, 2nd Edition by Geir Evensen.
+ *
+ * Eq. 14.34 is as follows:
+ *
+ *     X2 = (I + \Lambda_1)^{-1/2} @ X^T_1 * S
+ *
+ * , but this function seems to implement the following:
+ *
+ *     X2 = X^T_1 * S
+ *     X2 = (\Lambda^{-1/2}_1 * X2^T)^T
```

Comment:
The interchange of `^{-1/2}` and `_1` compared to line 44, does that carry any meaning here?

Reply:
In line 44 we do take the square root of (I + \Lambda_1), while the code does takes the square root of just \Lambda.
Does that maybe answer your question?

Reply:
No biggie, I think I would have written `\Lambda_1^{-1/2}` in line 49. In LaTeX it is rendered identically, but a tad less confusing for the ascii reading human.

Reply:
I cater to non-ascii readers only :)